test/extended/oauth/helpers.go

```diff
@@ -0,0 +1,37 @@
+package oauth
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+
+func waitForAuthOperatorAvailable(oc *exutil.CLI) error {
```

Comment:
this is going to be commonly used.  The easy way out is to simply pass in your own readiness function.

How is this different than a custom function to `UntilWithSync` ?